packages/mds-policy-service/repository/entities/policy-metadata-entity.ts

```diff
@@ -32,6 +33,9 @@ export class PolicyMetadataEntity extends IdentityColumn(class {}) implements Po
   @JoinColumn({ name: 'policy_id' })
   policy_id: PolicyMetadataEntityModel['policy_id']
 
+  @Column('varchar', { length: 32, nullable: true })
```

Comment:
You may be able to avoid checking for template changes in `updatePolicyMetadata` by using `readonly: true` here. But do we need to allow unpublished policies to change templates?

Reply:
Good question. This was part of the acceptance criteria:

    2. Once a policy has been published, we should not permit changes to the `template` field. 

But you're right. Once the metadata is created, it shouldn't be updatable. Asking to change acceptance criteria.

Reply:
See 72bbf98 for changes.